tests/test_example_dags.py

```diff
@@ -33,7 +33,7 @@
     "2.8": ["cosmos_manifest_example.py", "simple_dag_async.py", "cosmos_callback_dag.py"],
 }
 
-IGNORED_DAG_FILES = ["performance_dag.py", "jaffle_shop_kubernetes.py"]
+IGNORED_DAG_FILES = ["performance_dag.py", "jaffle_shop_kubernetes.py", "cosmos_callback_dag.py"]
```

Comment:
Removed just to speed up tests - will re-add this DAG before merging this PR
```suggestion
IGNORED_DAG_FILES = ["performance_dag.py", "jaffle_shop_kubernetes.py"]
```